_schemaV2.graphql

```diff
@@ -1329,6 +1329,9 @@ type Artist implements EntityWithFilterArtworksConnectionInterface & Node & Sear
     earliestCreatedYear: Int
     first: Int
 
+    # Include upcoming auction results
+    includeUpcoming: Boolean = true
```

Comment:
Do we want by default to have included upcoming?

Reply:
yes, was going to make this change right now